patroni/api.py

```diff
@@ -1263,6 +1263,7 @@ def get_postgresql_status(self, retry: bool = False) -> Dict[str, Any]:
             row = self.query(stmt.format(postgresql.wal_name, postgresql.lsn_name,
                                          postgresql.wal_flush), retry=retry)[0]
             result = {
+                'name': postgresql.name,
```

Comment:
If we go this way maybe we should also add the `scope`?

Reply:
I'm not sure what you are mentioning. I'm chasing the "name" in the output of the "/patroni". Let's do it with a separate PR